assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -125,7 +132,9 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
         refreshPostEditURL(storyId);
       })
       .catch(() => {
-        // TODO Display error message to user as save as failed.
+        showSnackbar({
+          message: __('Failed to save the story', 'web-stories'),
```

Comment:
Should we include some extra technical information? If a superuser wants to try to do something about it, they need more info. Or if any of our alpha/beta testers come across this, they need more information in order to file a better bug report.

Can we add extra info to the snack bar, that can be expanded, or something like that? Something like a _"Show technical information"_ toggle?

Reply:
What like this https://github.com/WordPress/gutenberg/pull/15657 ?

Reply:
I also think that we can follow up on this after.